accumulo/src/test/java/org/vertexium/accumulo/AccumuloResource.java

```diff
@@ -0,0 +1,148 @@
+package org.vertexium.accumulo;
+
+import org.apache.accumulo.core.client.AccumuloException;
+import org.apache.accumulo.core.client.AccumuloSecurityException;
+import org.apache.accumulo.core.client.Connector;
+import org.apache.accumulo.minicluster.MiniAccumuloCluster;
+import org.apache.accumulo.minicluster.MiniAccumuloConfig;
+import org.junit.rules.ExternalResource;
+import org.vertexium.GraphConfiguration;
+import org.vertexium.test.GraphTestBase;
+import org.vertexium.util.VertexiumLogger;
+import org.vertexium.util.VertexiumLoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.vertexium.test.GraphTestBase.*;
+
+public class AccumuloResource extends ExternalResource {
```

Comment:
Using the ClassRules seems to be a good way to pull out common BeforeClass/AfterClass and Before/After behavior from tests. I haven't played with them at all, but could we use [JUnit 5 dynamic tests](http://junit.org/junit5/docs/current/user-guide/#writing-tests-dynamic-tests) to somehow break the graph tests in to parts that can be included at test time?